src/Components/Navbar/Navbar.jsx

```diff
@@ -1,8 +1,8 @@
-import { useState } from "react";
+import React, { useState } from "react";
```

Comment:
No need to import React because of new JSX Transform introduced in React 17. 

[Reference [1]](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html)

[Reference [2]](https://dev.to/titungdup/you-no-longer-need-to-import-react-from-react-3pbj)